### ISSUE TITLE
Customize annotation IDs

### ISSUE THREAD

**Issue by waahlstrand** [heartexlabs/label-studio#2192]
**Is your feature request related to a problem? Please describe.**
Is it currently possible to customize the format of the automatic annotation IDs? The current random strings, although very unique, are not very human readable and referable. 

**Describe the solution you'd like**
Preferably, it would be nice to have support for different kinds of IDs, for example based on the annotation type (e.g. class, entity) or based on the task (e.g. `task <task id>:<annotation id>`) or the like.

**Describe alternatives you've considered**
Most importantly, I would love to know what parts of the code governs this to address it myself or modify it locally.

**Comment by makseq** [comment]
1. Are you talking about annotatoin.id or annotation.result[N].id? annotation.result[N].id is a really random string. And you can change its generation in a not difficult way. 
2. The annotation.id generation can't be changed, it's provided by DB automatically. It's a not string, it's `int` always.

**Comment by waahlstrand** [comment]
No, that's correct, I don't refer to the primary key of the annotations in the db, but rather for example the region ids. 

Is there an easy way to change the generation of annotation.result[N].id?
